modules/document_repository/php/files.class.inc

```diff
@@ -348,6 +348,7 @@ class Files extends \NDB_Page
         $uploadedFile = $uploadedFiles['file'];
         $fileSize     = $uploadedFile->getSize();
         $fileName     = $uploadedFile->getClientFileName();
+        $fileName     = urldecode($fileName);
```

Comment:
```suggestion
        // urldecode() necessary to decode double quotes encoded automatically 
        // by chrome browsers to avoid XSS attacks
        $fileName     = urldecode($fileName);
```